packages/management/addons-manager/bundle/vendir.lock.yml

```diff
@@ -0,0 +1,11 @@
+apiVersion: vendir.k14s.io/v1alpha1
+directories:
+- contents:
+  - manual: {}
+    path: upstream
+  - manual: {}
+    path: overlays
+  - manual: {}
+    path: values.yaml
+  path: 1.4.0/bundle/config
```

Comment:
can you re-generate this vendir lock file, you can run `make management-package-vendir-sync` to do that

Reply:
thanks @yharish991 , I have updated the vendir.yml path and ran make management-package-vendir-sync again to update vendir.lock.yml